### PR TITLE
:seedling: Webui enhancements

### PR DESCRIPTION
**Pull request by mudler**
**What this PR does / why we need it**: 

Currently the WebUI doesn't start on Alpine and this PR fixes it. This also avoids the tty to loop or stall if no provider is found, now it will drop to a shell by default and print the WebUI address and the available interfaces.

### CODE REVIEW

File: internal/agent/install.go [hunk 1]
```diff
@@ -110,14 +133,24 @@ func Install(dir ...string) error {
...
 	}
 
-	_, err = bus.Manager.Publish(events.EventChallenge, events.EventPayload{Config: cc.String()})
+	agentConfig, err := LoadConfig()
 	if err != nil {
 		return err
 	}
 
+	// try to clear screen
+	fmt.Print("\033c")
```
Comment (mauromorales) [inline internal/agent/install.go:142]: would be nice to have these kind of comment + expression as functions part of a Kairos library, for readability and reuse

File: internal/agent/install.go [hunk 2]
```diff
@@ -110,14 +133,24 @@ func Install(dir ...string) error {
...
+
 	cmd.PrintBranding(DefaultBanner)
 
-	agentConfig, err := LoadConfig()
+	// If there are no providers registered, we enter a shell for manual installation and print information about
+	// the webUI
+	if len(bus.Manager.Plugins) == 0 {
```
Comment (mauromorales) [inline internal/agent/install.go:148]: I'd recommend to start using predicate functions for readability and flexibility for future changes. For example something like `bus.Manager.HasRegisteredProviders()`

File: pkg/config/config.go [hunk 3]
```diff
@@ -19,6 +19,8 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const DefaultWebUIAddr = ":8080"
```
Comment (mauromorales) [inline pkg/config/config.go:22]:
```suggestion
const DefaultWebUIPort = ":8080"
```

File: internal/agent/install.go [hunk 4]
```diff
@@ -38,6 +39,28 @@ func optsToArgs(options map[string]string) (res []string) {
 	return
 }
 
+func displayInfo(agentConfig *Config) {
+	fmt.Println("--------------------------")
+	fmt.Println("No providers found, dropping to a shell. \n -- For instructions on how to install manually, see: https://kairos.io/docs/installation/manual/")
+	if !agentConfig.WebUI.Disable {
+		if agentConfig.WebUI.ListenAddress == "" {
```
Comment (mauromorales) [inline internal/agent/install.go:46]: also could be a predicate funcion

File: internal/agent/install.go (hunk 2 above)
Comment (mudler) [reply internal/agent/install.go]: good catch! addressed in https://github.com/kairos-io/kairos/pull/620/commits/8ae581024504ea5a84c730c547cbaa2c22e01d17 !

File: pkg/config/config.go (hunk 3 above)
Comment (mudler) [reply pkg/config/config.go]: err, changed to WebUIListenAddr. it includes also the bind addr, e.g. "127.0.0.1:8080" is valid input

**Review by mauromorales — APPROVED** [review]
lgtm

**Pull request merged**